Faction audit3

docofmur opened the pull request:
Removed redundant factions
checked all factions for transports - added where needed
added mod aircraft where needed under the new mod selection system

docofmur commented:
there were weird conflicts in 2 of the files, I fixed the typo's but they weren't in my files. France 1995 and France 2005, just putting them here in case it breaks something I know which ones

DanAlbert commented:
It's not possible to submit a PR that has conflicts, btw.

docofmur commented:
no i'll just try to keep my own notes that was just incase I had somehow messed up a faction again I knew which ones because i don't know why I got those conflict errors, they weren't in my files.

DanAlbert commented:
My mistake, there are no conflicts; just conflicts with the merge strategy I had selected on my end.

DanAlbert commented on resources/factions/russia_1975.json:
```diff
@@ -7,7 +7,9 @@
   "aircrafts": [
+    "IL-76MD",
     "Mi-24V Hind-E",
+    "Mi-24P Hind-F",
```
This was intentional (as was adding the new "Russia 1975 Mi-24P" faction). This aircraft wasn't used in 1975. The difference is historical accuracy for opfor vs flyability for players, not mod support.

DanAlbert replied:
https://github.com/dcs-liberation/dcs_liberation/pull/1267

docofmur replied:
ok it really does seem weird to keep them both since only one is flyable but ok, i'm assuming your adding it back in and I don't need to add it back in. Should I audit our factions for accuracy then, cause I didn't do that

DanAlbert replied:
> Should I audit our factions for accuracy then, cause I didn't do that

If you want. The author of the PR did but I don't think it's urgent.

docofmur committed:
Faction Audit.

Transports and mod aircraft added where needed cleaned up various
duplicates in factions.

DanAlbert commented on resources/factions/japan_2005.json:
```diff
@@ -9,8 +9,9 @@
+    "C-130",
+    "C-130J-30 Super Hercules",
     "F-15J Eagle",
-    "F-2A",
```
Not sure why this was deleted? I'm adding it back because it looks unintentional.
